### PR TITLE
Launcher generates unlimited runner IDs

### PR DESCRIPTION
**Pull request by gronke**
## Proposed changes

When running more than 26 suites on a single instance, the rid pattern turns from letters to special characters (`y`, `z`, `{`, `|`, `}`, ...). A problem occurs when reporters like wdio-junit-reporter include that identifier in an output file name.

Here is an example that outlines the problem:
```
> var rid = 'a'
undefined
> rid = String.fromCharCode(rid.charCodeAt(0)+1)
'b'
> rid = String.fromCharCode(rid.charCodeAt(0)+1)
'c'
> rid = 'z'
'z'
> rid = String.fromCharCode(rid.charCodeAt(0)+1)
'{'
> rid = String.fromCharCode(rid.charCodeAt(0)+1)
'|'
> rid = String.fromCharCode(rid.charCodeAt(0)+1)
'}'
> rid = String.fromCharCode(rid.charCodeAt(0)+1)
'~'
> rid = String.fromCharCode(rid.charCodeAt(0)+1)
''
> rid = String.fromCharCode(rid.charCodeAt(0)+1)
'
```

With the changes from this pull-request both the cid and the rid are numeric values separated by a delimiter.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by monolithed** [comment]
👍 it seems my issue https://github.com/webdriverio/wdio-junit-reporter/issues/19 can be closed

**Comment by gronke** [comment]
@monolithed Yes, that bug would be closed with this changes. 

I'm not sure though if something depended on the old way of counting rids, but in all my scenarios counting numbers works well too.

File: lib/launcher.js [hunk 1]
```diff
@@ -320,13 +319,11 @@ class Launcher {
      * @return {String}     runner id (combination of cid and test id e.g. 0a, 0b, 1a, 1b ...)
```
Comment (christian-bromann) [inline lib/launcher.js]: can we update this comment?

**Comment by christian-bromann** [comment]
Fixed outstanding issue by myself as I want to move forward with the release.

**Pull request closed without merging**